ydb/docs/en/core/integrations/orm/jooq.md

```diff
@@ -0,0 +1,187 @@
+- `strategy.name`: `tech.ydb.jooq.codegen.YdbGeneratorStrategy` (recommended setting)
+
+Let's look at an example of the `maven` plugin:
+
+```xml
+<plugin>
+    <groupId>org.jooq</groupId>
+    <artifactId>jooq-codegen-maven</artifactId>
+    <version>3.19.11</version>
+    <executions>
+        <execution>
+            <goals>
+                <goal>generate</goal>
+            </goals>
+        </execution>
+    </executions>
+    <dependencies>
+        <dependency>
+            <groupId>tech.ydb.jdbc</groupId>
+            <artifactId>ydb-jdbc-driver</artifactId>
+            <version>${ydb.jdbc.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>tech.ydb.dialects</groupId>
+            <artifactId>jooq-ydb-dialect</artifactId>
+            <version>${jooq.ydb.version}</version>
+        </dependency>
+    </dependencies>
+    <configuration>
+        <jdbc>
+            <driver>tech.ydb.jdbc.YdbDriver</driver>
+            <url>jdbc:ydb:grpc://localhost:2136/local</url>
+        </jdbc>
+        <generator>
+            <strategy>
+                <name>tech.ydb.jooq.codegen.YdbGeneratorStrategy</name>
+            </strategy>
+            <database>
+                <name>tech.ydb.jooq.codegen.YdbDatabase</name>
+                <!-- исключение системных таблицы -->
+                <excludes>.sys.*</excludes>
+            </database>
+            <target>
+                <packageName>ydb</packageName>
+                <directory>./src/main/java</directory>
+            </target>
+        </generator>
+    </configuration>
+</plugin>
+```
+
+Example of generated classes from [YQL tutorial](../../dev/yql-tutorial/create_demo_tables.md):
+
+```
+ydb/DefaultCatalog.java
+ydb/default_schema
+ydb/default_schema/tables
+ydb/default_schema/tables/Seasons.java
+ydb/default_schema/tables/records
+ydb/default_schema/tables/records/SeriesRecord.java
+ydb/default_schema/tables/records/EpisodesRecord.java
+ydb/default_schema/tables/records/SeasonsRecord.java
+ydb/default_schema/tables/Series.java
+ydb/default_schema/tables/Episodes.java
+ydb/default_schema/Indexes.java
+ydb/default_schema/Keys.java
+ydb/default_schema/Tables.java
+ydb/default_schema/DefaultSchema.java
+```
+
+## Usage {#usage}
+
+To integrate {{ ydb-short-name }} with JOOQ into your project, you need to add two dependencies: {{ ydb-short-name }} JDBC Driver and the JOOQ extension for {{ ydb-short-name }}.
+
+Examples for different build systems:
+
+{% list tabs %}
+
+- Maven
+
+    ```xml
+    <!-- Set actual versions -->
+    <dependency>
+        <groupId>tech.ydb.jdbc</groupId>
+        <artifactId>ydb-jdbc-driver</artifactId>
+        <version>${ydb.jdbc.version}</version>
+    </dependency>
+
+    <dependency>
+        <groupId>tech.ydb.dialects</groupId>
+        <artifactId>spring-data-jdbc-ydb</artifactId>
+        <version>${spring.data.jdbc.ydb}</version> 
+    </dependency>
+    ```
```

Comment:
There was a similar example in lines 16-60: why do you give a complete example first and then repeat a small part of it?